docs/contributing/contributing_setup.md

```diff
@@ -58,7 +58,7 @@ source ge_dev/bin/activate
 
 pip install --upgrade pip==21.3.1
 
-pip install -r requirements-dev-lite.txt -c constraints-dev.txt -e .
+pip install -r requirements.txt -r requirements-dev-lite.txt -c constraints-dev.txt -e .
```

Comment:
Is this the only instance we need to change in the docs?

Reply:
Yea, when I first added that "easy" section to the contributing doc, the dev-lite file initially had a `--requirement requirements.txt` at the top. This is the only place in docs where the `requirements-dev-lite.txt` file is referenced